docker_files/recipes/python/meta/README.md

```diff
@@ -0,0 +1,33 @@
+# Python 2.7
+This package provides core Python 2.7 runtime environment. It includes default builtin modules.
+
+## Usage
+Run Python shell:
+```
+$ capstan run python --boot shell
```

Comment:
If you decide to rename configuration from `shell` into `python`, please do it here as well.